api/grpcserver/node_service.go

```diff
@@ -106,6 +106,30 @@ func (s NodeService) NodeInfo(context.Context, *empty.Empty) (*pb.NodeInfoRespon
 	}, nil
 }
 
+func (s NodeService) PeerInfo(context.Context, *empty.Empty) (*pb.PeerInfoResponse, error) {
```

Comment:
better to use AdminService for this. we don't set NodeService to 127.0.0.1 by default, i would prefer to not expose this info accidentally